feature-libs/checkout/base/components/checkout-delivery-address/checkout-delivery-address.component.html

```diff
@@ -1,104 +1,106 @@
-<h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
-  {{ 'checkoutAddress.shippingAddress' | cxTranslate }}
-</h2>
+<section [attr.aria-label]="'checkoutAddress.shippingAddress' | cxTranslate">
```

Comment:
Unfortunately, I think changing the structure here would be breaking.

Reply:
Hi @Pio-Bar! Thanks for your comment 👍 . Fixed